components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -30,11 +30,14 @@ Creating default teams Editors and Viewers...
 Success: Enabled IAM v2
 ```
 
+Note: To upgrade without porting existing policies, use the `--skip-policy-migration` flag: `chef-automate iam upgrade-to-v2 --skip-policy-migration`.
+
 ## View Policies
 
 After you've logged in to Chef Automate, select the **Settings** tab in the top navigation bar, then select and locate the `Policies` section on the left hand panel.
 
-This lists all of your v2 policies:
+Here you can view all of your v2 policies. If you have upgraded without using the `--skip-policy-migration` flag, you will
```

Comment:
```suggestion
In this section, you can view all of your v2 policies. If you have upgraded without using the `--skip-policy-migration` flag, you will
```

Reply:
Also, ideally, it would be great if lines 39 and 40 were put together rather than suddenly split mid-sentence.